harvard-lts/mps-viewer
# Upgrade mirador-share-plugin to 0.16.0

**Upgrade mirador-share-plugin to 0.16.0**
* * *

**JIRA Ticket**: [LTSVIEWER-272](https://jira.huit.harvard.edu/browse/LTSVIEWER-272)

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
   * PR with the plugin work: https://github.com/ProjectMirador/mirador-share-plugin/pull/49

# What does this Pull Request do?
Updates the "Alternate viewer" section with a clear and easy alternative to the cursor click and drag functionality. Also adds a visual and aural cue that the "Copy" button has been clicked (applied to all copy to clipboard buttons).

These changes have already been built to npmjs as [@mirador-share-plugin 0.16.0](https://www.npmjs.com/package/mirador-share-plugin).

# How should this be tested?
* Spin up viewer from this branch
* Open the "Share" modal window (a dropdown option under the three dots in the top right of viewer)
* Visually check to see that the share modal window matches that shown in the screenshot below
* Click on each "copy" button and see that a notification window is triggered in the top center (it should have a green background, with a success icon and message that says "Link/Code/Manifest copied to clipboard!")
   * The "Share link" copy to clipboard button now points to the full page viewer (was previously pointing to the manifest)
   * The new "Add to another viewer" copy to clipboard button points to the manifest
* Click and drag the IIIF icon into another viewer instance to ensure that functionality is still working and opens up the image in the new viewer space
   * The cursor has also been changed from a "pointer" to a hand "grab"
* Click on the "What is IIIF?" link to ensure that is still working and brings you to the IIIF.io homepage.


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @f8f8ff

# Share modal window screenshot
![](https://user-images.githubusercontent.com/13121501/272965531-654d759e-fa1d-4432-9a06-52d0c57eea11.png)
